test/pgcdc/mzcompose

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I have a complete test that uses the new approach ready, so please do not push the stuff in ```test/pgcdc```